### ISSUE TITLE
App Doesn't Pass Integrity Check

### ISSUE THREAD

**Issue by paulcalabro** [mtierltd/timetracker#49]
The following error message is displayed in the "Security & setup warnings" section of settings:

```
Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- timetracker
	- FILE_MISSING
		- build/artifacts/appstore/timetracker.tar.gz

Raw output
==========
Array
(
    [timetracker] => Array
        (
            [FILE_MISSING] => Array
                (
                    [build/artifacts/appstore/timetracker.tar.gz] => Array
                        (
                            [expected] => e3b6e994bacea5de17848ed974244d9880e85113846233e0a53de30cc1aeff26908f7a53eba05f88545b5878716db118132601a0db7033a75caec373a22675b3
                            [current] => 
                        )

                )

        )

)

```

**Comment by ovpc** [comment]
Confirm time tracker fails intergrity check when updating NC19.0.3 to 19.0.4
Let me know if need other logs.
>Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- timetracker
	- FILE_MISSING
		- build/artifacts/appstore/timetracker.tar.gz

Raw output
==========
Array
(
    [timetracker] => Array
        (
            [FILE_MISSING] => Array
                (
                    [build/artifacts/appstore/timetracker.tar.gz] => Array
                        (
                            [expected] => e3b6e994bacea5de17848ed974244d9880e85113846233e0a53de30cc1aeff26908f7a53eba05f88545b5878716db118132601a0db7033a75caec373a22675b3
                            [current] => 
                        )

                )

        )

)

**Comment by ovpc** [comment]
Thank you fixed in 0.0.45
